prometheus-ksonnet/grafana/dashboards.libsonnet

```diff
@@ -10,17 +10,48 @@
   // New API: Mixins go in the mixins map.
   mixins+:: {},
 
-  // emptyMixin allows us to reliably do `mixin.grafanaDashboards` without
+  // mixinProto allows us to reliably do `mixin.grafanaDashboards` without
   // having to check the field exists first. Some mixins don't declare all
   // the fields, and thats fine.
-  local emptyMixin = {
-    grafanaDashboards+: {},
+  //
+  // We also use this to add a little "opinion":
+  // - Dashboard UIDs should be the md5 hash of their filename.
+  // - Timezone should be "default" (ie local).
+  // - Tooltip should only show a single value.
```

Comment:
`should` sounds like we do validation here – but we actually just force this behaviour